roles/docker-app/defaults/main.yml

```diff
@@ -3,3 +3,4 @@ docker_app_env: []
 docker_app_ports: []
 docker_app_extra_params: ""
 docker_app_syslog_identifier: "{{ docker_app_name }}"
+docker_app_restart_sec: "10s"
```

Comment:
Can this be something else than `XXs`? if the unit is part of the string value then sec is misleading if it can be `500ms`

Reply:
Good point! I'll rename it to docker_app_restart_delay.